A24Group/ssGWT-lib
# Added spinner component for the dynamic form

Added spinner for dynamic form

Test
- Check out
- Compile jar
- Copy jar to triage
- cap a24deploy:gwt:application_compile
- Make sure nothing breaks

Plz review
- Note dont review the classes added by Google. See license 

A24Group/Triage#3322
